src/utils/bytes.js

```diff
@@ -32,3 +32,28 @@ export function stringToBytes(str) {
   }
   return bytes;
 };
+
+/**
+ * Converts a 8-bit bytes array into a string
+ * @param {!Uint8Array} bytes
+ * @return {string}
+ */
+export function bytesToString(bytes) {
+  return String.fromCharCode.apply(String, bytes);
+};
+
+/**
+ * Generate a random bytes array with specific length
+ * @param {!number} length
+ * @return {!Uint8Array}
+ */
+export function getRandomBytesArray(length) {
```

Comment:
> hmm, what i learned from security folks is that all consequent Math.random calls are predictable, which means no extra entropy. So it doesn't really give you more bytes.

That's the "psuedo" part. But it's still much more random than what we're doing here. https://medium.com/@betable/tifu-by-using-math-random-f1c308c4fd9d#.zi48jhb65